.bazelrc

```diff
@@ -0,0 +1 @@
+common --all_incompatible_changes
```

Comment:
To be clear, adding a .bazelrc to this repo shouldn't have any effect on users who import this repo to their own workspace? I can't imagine that it would, but if it did that'd be a show-stopper since we would be breaking downstream users.

Reply:
That is my understanding, yes. This will not affect users who import Skylib into their workspace.

The advantage of doing this is for users who do use `--all_incompatible_changes` in their workspace and import Skylib. If Skylib does not build with `--all_incompatible_changes`, then their repo will fail to build. That's the motivation for this change.